fe/fe-common/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -834,6 +834,10 @@ public static boolean isImplicitlyCastable(Type t1, Type t2, boolean strict, boo
     }
 
     public static boolean canCastTo(Type sourceType, Type targetType) {
+        // In BE Code , we make any data type can cast to jsonb
+        if (targetType.isJsonbType()) {
+            return true;
```

Comment:
agg_state, variant could cast to json? what we will get if cast ipv4 /ipv6 to json.